src/20240415_AaveV3Ethereum_ChaosLabsEngagementAmendment/AaveV3Ethereum_ChaosLabsEngagementAmendment_20240415.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+/**
+ * @title Chaos Labs Engagement Amendment
+ * @author Chaos Labs
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x34b6cf5bc9c8a0525c5b32d4ce2ca2ccfce39d15bd0aba5cab46a4e9e78f3ea8
+ * - Discussion: https://governance.aave.com/t/arfc-chaos-labs-engagement-amendment/17324
+ */
+contract AaveV3Ethereum_ChaosLabsEngagementAmendment_20240415 is IProposalGenericExecutor {
+  address public constant CHAOS_LABS_TREASURY = 0xbC540e0729B732fb14afA240aA5A047aE9ba7dF0;
+  uint256 public constant STREAM_AMOUNT_GHO = 400_000 ether;
+  uint256 public constant STREAM_DURATION = 200 days;
+
```

Comment:
```suggestion
  uint256 public constant ACTUAL_STREAM_AMOUNT =
    (STREAM_AMOUNT_GHO / STREAM_DURATION) * STREAM_DURATION;
    
```

stream amount should always be fully claimable, therefore the amount must be exactly divisible by emissionPerSecond

the final amount will be a few wei less due to rounding: `399999999999999997440000`